contract-artifacts/version-config.json

```diff
@@ -1,3 +1,3 @@
 {
-  "gateway": "v3.0.0"
+  "gateway": "v3.1.0"
```

Comment:
Shouldn't do this in this PR that's not related. It's updated in https://github.com/axelarnetwork/axelar-core/pull/1416, and won't work here.